<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subjectWindow</class>
 <widget class="QWidget" name="subjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Тип&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>75</number>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Назва&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Скасувати</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainImage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Оберіть тип</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Фізична культура і здоровя</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Навчальний предмет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Антропологічні науки</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Суспільствознавство</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Природознавство</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Технічні науки</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Математика</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Профільний</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Космологія</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="semestrLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Семестровий контроль&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="controlComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Оберіть семестровий контроль</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Залік</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Іспит</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="allTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="allTimeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Всього годин&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lektsiaLabel">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>290</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Лекційні години&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="lectureTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>310</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="labTimeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Лабораторних годин&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="labTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="seminarSpinBox">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>360</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="semestrTimeLabel">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>340</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Семінарних годин&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="soloWorkSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="soloTimeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Самостіних робот годин&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="semestLearnLabel">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>390</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Семестр вивчення&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="semesrtLearnSpinBox">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>410</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="okLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>125</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%status_label%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
